ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
I think it's just a little bit like Taiwanese English, mind to think about how to improve it again?